pubspec.yaml

```diff
@@ -0,0 +1,18 @@
+name: centrifuge_dart
```

Comment:
Is it convenient in Dart to call repos like `centrifuge-dart` i.e. using minus sign as separator?

Reply:
Maybe it's even better to call it just `centrifuge` - this is for example how Javascript client named